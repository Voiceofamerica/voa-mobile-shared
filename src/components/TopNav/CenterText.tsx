
import * as React from 'react'

import { centerText } from './TopNav.scss'

export interface Props extends React.Props<any> {
}

function CenterText (props: Props) {
  const { children } = props

  return (
    <div className={centerText}>
      {children}
    </div>
  )
}

export default CenterText
