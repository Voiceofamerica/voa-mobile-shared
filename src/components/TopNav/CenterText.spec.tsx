
import * as React from 'react'
import { create } from 'react-test-renderer'

import CenterText from './CenterText'

describe('<CenterText />', () => {
  describe('shapshots', () => {
    it('should render empty', () => {
      const element = create((
        <CenterText />
      ))

      expect(element).toMatchSnapshot()
    })

    it('should render with the given children', () => {
      const element = create((
        <CenterText>
          <div>child1</div>
          <div>child2</div>
        </CenterText>
      ))

      expect(element).toMatchSnapshot()
    })
  })
})
