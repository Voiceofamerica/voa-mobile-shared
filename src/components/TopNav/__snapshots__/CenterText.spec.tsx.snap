// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CenterText /> shapshots should render empty 1`] = `
<div
  className="centerText"
/>
`;

exports[`<CenterText /> shapshots should render with the given children 1`] = `
<div
  className="centerText"
>
  <div>
    child1
  </div>
  <div>
    child2
  </div>
</div>
`;
